DashAI/back/api/api_v1/api.py

```diff
@@ -1,6 +1,7 @@
 from fastapi import APIRouter
 
-from DashAI.back.api.api_v1.endpoints import datasets
+from DashAI.back.api.api_v1.endpoints import datasets, tasks
 
 api_router_v1 = APIRouter()
 api_router_v1.include_router(datasets.router, prefix="/dataset")
+api_router_v1.include_router(tasks.router, prefix="/task")
```

Comment:
There is a conflict here with recent push on staging branch, please fix it (merge staging and push the fix)